services/core-web/src/components/noticeOfWork/applications/NOWDocuments.js

```diff
@@ -161,6 +174,49 @@ export const NOWDocuments = (props) => {
       tableColumns.splice(2, 0, descriptionColumn);
     }
 
+    const deleteButtonColumn = {
+      title: "",
+      dataIndex: "isDeletionAllowed",
+      key: "isDeletionAllowed",
+      render: (isDeletionAllowed, record) => {
+        if (isDeletionAllowed) {
+          return (
+            <Popconfirm
```

Comment:
Can you wrap all the delete actions into a <NOWActionWrapper  /> with the correct permissions and tabs - this will allow them to be disabled based off the status of the tab/application

Reply:
Thanks! it is done.